src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+/**
+ * An extension of the Exception class used to identify exceptions unique to Duke.
+ */
+public class DukeException extends Exception {
```

Comment:
Could it be better to have different types of DukeException instead? With this way of implementation all error messages would be in this file, and can be edited without going into other files unnecessarily. (encapsulation)